### ISSUE TITLE
Document required permissions

### ISSUE THREAD

**Issue by adrianjost** [CodelyTV/pr-size-labeler#47]
It took me a bit of time to figure out which [permissions](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#overview) this actions needs.

In my case, adding the following worked. Maybe that should be added to the README.

```yml
jobs:
  labeler:
    ...
    permissions:
      issues: write
      pull-requests: write
    steps:
      ...
```

**Comment by EngJay** [comment]
+1 - after removing the github url per #40, the action runs without indication that the lack of permissions is preventing it from labeling PRs, which led me here.

**Comment by JavierCane** [comment]
Good one. Thanks for opening the issue and providing the actual fix for it.

you can expect we work on this in the upcoming days/weeks at most. If you want, we would really appreciate opening a PR modifying the current README.md in order to reflect this by default and we would merge it even sooner.

Thanks!

**Comment by adrianjost** [comment]
I didn't open a PR yet, because I've used the permission stuff myself for the first time here. So I don't feel comfortable writing any documentation regarding that. I don't even know if any other potential defaults might affect my solution, so it only works in my environment and in others more permissions would be needed. I have no idea.

My main intent for this Issue was, to provide some information in case someone struggles with the same setup issue and hoping that someone who actually understands what I did would write some documentation eventually.
